PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/arm/action/UpdateDeploymentAction.java

```diff
@@ -33,8 +36,10 @@ public UpdateDeploymentAction(DeploymentNode deploymentNode) {
     @Override
     protected void actionPerformed(NodeActionEvent nodeActionEvent) {
         Project project = (Project) deploymentNode.getProject();
-        AzureSignInAction.doSignIn(AuthMethodManager.getInstance(), project).subscribe((isSuccess) -> {
-            this.doActionPerformed(nodeActionEvent, isSuccess, project);
+        AzureSignInAction.signInIfNotSignedIn(project).subscribe((isLoggedIn) -> {
+            if (isLoggedIn && AzureLoginHelper.isAzureSubsAvailableOrReportError(message("common.error.signIn"))) {
+                AzureTaskManager.getInstance().runLater(() -> this.doActionPerformed(nodeActionEvent, isLoggedIn, project));
+            }
```

Comment:
AzureSignInAction.requireSignedIn(()->this.doActionPerformed(nodeActionEvent, true, project))